core/src/tile/tile.h

```diff
@@ -51,7 +49,7 @@ class Tile {
     float getScale() const { return m_scale; }
 
     /* Returns the reciprocal of <getScale()> */
-    float getInverseScale() const { return m_inverseScale; }
+    float getInverseScale() const { return 1.0f / m_scale; }
```

Comment:
I would still store the `inverse scale` as a member. Doing an division repeatedly is usually not recommended, specially a floating-point division.